ui/src/pages/author-guide.tsx

```diff
@@ -50,332 +334,30 @@ const AuthorGuide: NextPage = (): React.ReactElement => (
-                                    made post-publication.
-                                </li>
-                                <li className="mb-6">
-                                    <span className="font-bold">What is the acceptance process for publications?</span>{' '}
-                                    There is no acceptance or rejection from Octopus – you are recording the work as
-                                    yours, and registering when you did so. Others can then assess the work, review it,
-                                    and build on it. You should therefore try to ensure that you are completely happy
-                                    with the work before you press &apos;publish&apos;. As soon as the last co-author
-                                    has agreed it, the work will be live and a DOI available.
-                                </li>
-                                <li className="mb-6">
-                                    <span className="font-bold">What if I made a mistake?</span> Octopus will allow for
-                                    reversioning of publications, but this feature is currently under development. When
-                                    released, reversioning is intended for cases where an author makes a mistake, or a
-                                    reviewer points out something which needs to be added or corrected. In the latter
-                                    case, we strongly recommend including the reviewer as a co-author on a new version
-                                    to recognise their input. All older versions remain visible on the record, but new
-                                    versions have their own DOI. All reviews and red flags remain with the version they
-                                    were originally linked to. We recommend keeping reversioning to a minimum.
-                                </li>
-                            </ul>
-                        </StandardText>
-                        <StandardText>
-                            <>
-                                If you require further guidance, or would like to suggest any additional author
-                                materials, please contact the team at{' '}
-                                <Components.Link
-                                    href="mailto:help@jisc.ac.uk"
-                                    openNew={true}
-                                    className="rounded text-teal-600 outline-0 transition-colors duration-500 focus:ring-2 focus:ring-yellow-400 dark:text-teal-300"
-                                >
-                                    help@jisc.ac.uk.
-                                </Components.Link>
-                            </>
-                        </StandardText>
-                    </div>
-                </>
-            </PageSection>
+            <div className="container mx-auto px-8 pb-10 pt-10 lg:gap-4 lg:pt-20">
```

Comment:
Possible to do these with Articles and Sections rather than divs. Helps with accessibility (and generally looks better, divs are so 2010)

Reply:
I'd probably do this something like:

```
            <section className="...">
                <Components.PageTitle ... />
            </section>
            <article className="...">
                <aside className="...">
                    <Components.PageContentsSidebar ... />
                </aside>
                <section className="...">
                    {authorGuideSections.map((authorGuideSection) => (
                        <dl key={authorGuideSection.id} id={authorGuideSection.id} className="mx-auto pt-4 lg:w-10/12">
                            ...
                        </dl>
                    ))}
                </section>
            </article>
```

I've put in ... for brevity but it may have made things more confusing.. Also the `<dl />` may need extra styling to cope with the css applied here

Reply:
Agree that article + section would help, but I'm bearing in mind if elements are separated just for styling reasons it's better to use a div than a section (per MDN: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section). I think I'd go for:

```
        <article>
            <Components.PageTitle className="..." />
            <aside className="...">
                <Components.PageContentsSidebar ... />
            </aside>
            <div className="...">
                {authorGuideSections.map((authorGuideSection) => (
                    <section key={authorGuideSection.id} id={authorGuideSection.id} className="mx-auto pt-4 lg:w-10/12">
                            ...
                     </section>
                 ))}
            </div>
        </article>
```

If that looks okay? I'm not convinced a <dl> is strictly right either so I would get rid of that too.

Reply:
Yeah I saw that article after I posted this 🙃 Yeah agree keep the div for the styling.

`<dl />` is needed because of the nested `<dd>` and `<dt>` though. Unless you change them to `<p>` or similar.

Reply:
Sounds good - just pushed a commit with the markup restructured.